index.html

```diff
@@ -411,8 +412,69 @@ <h2 class="w-100">Contact me</h2>
 
             </section>
         </section>
-
     </div>
+    <!--here start the code for popup-->
+    <section class="popup-window grid-container fixed w-100 h-100 top-left bg-popup hidden">
+        <!--<div class="w-90 bg-white h-90 m-auto b-rounded p-4 overflow-scroll">
+            <div class="grid col-2">
+                <div>
+                    <h3 class="popup-title text-blue-custom text-bold p-0 m-0">Tonic</h3>
+                </div>
+                <div class="text-right center grid end">
+                    <img src="./images/close1.png" alt="image_cross"  width="30px"/>
+                </div>
+            </div>
+            <div>
+                <div class="card-sub-title w-90">
+                    <b>CANOPY</b>
+                    <span class="counter">
+                        <img src="./images/Counter.png" alt="counter">
+                        Back End Dev
+                    </span>
+                    <span class="counter">
+                        <img src="./images/Counter.png" alt="counter">
+                        2015
+                    </span>
```

Comment:
- [ ] Kindly remove all commented code blocks in the code. Commented code tends to make the project untidy, refer to GitHistory to use previous code verisons.